heroku/index.js

```diff
@@ -45,7 +45,19 @@ function createWebhook(smoochCore, target) {
         })
         .then((res) => {
             console.log('Smooch webhook created at target', res.webhook.target);
-        })
+            return smoochCore.webhooks.create({
```

Comment:
Nit: you only need one webhook. When the webhook fires the trigger type is included in the payload.